src/Core/Geometry/MeshPrimitives.cpp

```diff
@@ -90,6 +91,139 @@ TriangleMesh makeSharpBox( const Vector3& halfExts,
     return makeSharpBox( aabb, color, generateTexCoord );
 }
 
+MultiIndexedGeometry makeSharpBox2( const Aabb& aabb,
```

Comment:
Do we need to have 2 methods to build sharpbox ? I think that this one, quad-based shar-box, should replace the triangulated one.

Reply:
nope, will be done before merging of course.